src/cascade/executor/construct_model.py

```diff
@@ -190,6 +195,13 @@ def construct_model_random_effects(default_age_time, single_age_time, ev_setting
             # One smooth for all children when there isn't a child location.
             location = None
         model.random_effect[(smooth.rate, location)] = re_grid
+        random_effect_by_rate[smooth.rate].append(location)
+
+    for is_rate_consistent, locations in random_effect_by_rate.items():
```

Comment:
`is_rate_consistent` is a very confusing name for this since it's just the name of the rate I think?

Reply:
changed